pandas/tests/io/parser/test_encoding.py

```diff
@@ -299,17 +299,21 @@ def test_readcsv_memmap_utf8(all_parsers):
     tm.assert_frame_equal(df, dfr)
 
 
-def test_not_readable(all_parsers, request):
+@pytest.mark.usefixtures("pyarrow_xfail")
```

Comment:
@jbrockmendel  Is this needed? Already have  `pytest.mark.xfail` in the body.

Reply:
i think either works, but only need one.